jwt_verify_lib/verify.h

```diff
@@ -22,13 +22,25 @@ namespace google {
 namespace jwt_verify {
 
 /**
- * This function verifies JWT signature.
- * If verification failed, returns the failture reason.
+ * This function verifies JWT signature is valid and that it has not expired checking the "exp" and "nbf" claim against
```

Comment:
line is too long,  make sure it is within 80 chars.

I just submitted a script to check coding style.

Please run 
script/check-style

Reply:
Fixed with running the check-style script and rebase.